controllers/storagecluster/kms_resources.go

```diff
@@ -39,6 +39,29 @@ var (
 	_ kmsConfigMapValidateFunc = reachKMSProvider
 )
 
+type kmsResourceManager struct{}
+
+func (kmsRMgr *kmsResourceManager) ensureDeleted(r *StorageClusterReconciler, sc *ocsv1.StorageCluster) error {
```

Comment:
IMO we should add logs as well for errors and info

Reply:
Then we should change the signature of `ensureDeleted()` and `ensureCreated()` functions and add `logger` parameter as well, right?

Reply:
No, we have a logger in the StorageClusterReconciler

Reply:
I guess you missed this.

Reply:
Thanks for pointing @iamniting ... Updated